### ISSUE TITLE
create a "wrap" option for adding windows

### ISSUE THREAD

**Issue by GoogleCodeExporter** [wmdiem/euclid-wm#51]
```
When we add a window to the view, it would be nice to have a configurable 
setting to wrap to the next track, e.g., if more than a certain number of 
windows are already in the track, put it in the next track. 

Will need to iron out the expected behavior. 

One issue will be predictability, right now a new window always appears below 
the currently focused window. The proposed behavior will break that 
expectation. I think this is acceptable, the new window still gets focus (so 
the basic workflow is the same, you can move or use it right away), it seems 
like having a usable default layout for the new window is more important than 
having a predictable layout, layout is still predictable, although somewhat 
smarter, and finally, we can introduce it as a configuration. 

As for logic:
I'm inclined to default to trying to make the ratio tracks : conts = 1. 
something like 

if conts in track now = tracks on view -> if there is another track, with fewer 
conts, put it there; else put it in the current track. 
if conts in track now > tracks on view -> walk through other tracks looking for 
one with fewer conts, if you reach the end of the list, make a new track. 

So if you open successively windows, without positioning them you get:

1) 1cont
2) 2conts
3) 2conts 1cont
4) 2conts 2conts
5) 3conts 2conts
6) 3conts 3conts
7) 3conts 3conts 1cont

Basically there are two important states a track can be in: Pressure and Full; 
when the number of conts in a track == the number of tracks the track is under 
pressure, and if another track isn't, that is where the new window goes. 
A track is Full when conts >= tracks+1; by default a window never gets put into 
a full cont, and if necessary a new track will be made. 


```

Original issue reported on code.google.com by `wmd...@gmail.com` on 17 Dec 2013 at 7:49
